CorsixTH/Lua/map.lua

```diff
@@ -735,6 +735,15 @@ function Map:getParcelTileCount(parcel)
   return self.parcelTileCounts[parcel] or 0
 end
 
+function Map:checkMapPathFinder()
```

Comment:
Why is this function in `Map`?  It doesn't even use any data from the map (besides `app`).

Looks more like a world function to me.

That also solves the problem that world may not exist when the function is called.